0002_child_test.js

```diff
@@ -461,7 +458,7 @@ describe("Test add-new forms:", () => {
    function saveAndCheck(parent, child, field, isTopTab) {
       // cy.log(parent, child, field);
       save(parent, child, field, isTopTab || null);
-      cy.get(".webix_warn").find(".webix_button").filter(":visible").click({multiple: true, force: true});
+      cy.get(".webix_warn").find(".webix_button").filter(":visible").click({ multiple: true, force: true });
```

Comment:
you can do `cy.get("...").find("...").first().click({ force: true })`
In my opinion this is better than clicking multiple as that could have unintended consequences.

Reply:
In other cases it could, for this specific one we are trying to get any and all new data to show on the front end. 

But, it would be better if there was a data-cy instead of a 'warn' to point at, so I agree that this should be refactored